data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +518,1020 @@ mission "Unfettered Jump Drive Trading"
+				accept
+			label declining
+			action
+				"unfettered rep: Mera" --
+			`	She clench her fist, visibly angry for a moment, before calming down. "Alright, but it will take a while for me to get someone else to scan a Geocoris, and we're missing a significant opportunity."`
+			choice
+				`	"Okay, okay. I changed my mind."`
+					goto choose
+				`	"Not my problem."`
+					decline
+		log "People" "Mera" `An Unfettered engineer who is interested in ship designs of all kinds. She is easily excited by new technologies and ships, and seems to communicate partly through her bodily language.`
+	on decline
+		event "unfettered: analyzed the scans" 30 60
+	on abort
+		dialog `Mera hails you when she learns of you aborting this mission. "This will slow us down significantly, by a month at least."`
+		"unfettered rep: Mera" -= 2
+		event "unfettered: analyzed the scans" 30 60
+	on waypoint
+		dialog `You do not see the Geocoris that Mera mentioned; it's probably in a system nearby. You should land and depart when it arrives.`
+		set "unfettered: entered Ya Hai"
+	npc "scan cargo"
+		government "Hai"
+		to spawn
+			has "unfettered: entered Ya Hai"
+		personality target entering uninterested waiting
+		system "Ya Hai"
+		ship "Geocoris (Keystones)" "Strong Gold"
+		conversation
+			branch "didn't listen"
+				not "outfit (flagship installed): Value Detector"
+			`You have scanned the Geocoris. You may now return to <destination> with the scanner logs.`
+				decline
+			label "didn't listen"
+			`You have scanned the Geocoris, but you did not use a Value Detector to do so. The scans will be of no value to Mera. You have failed the mission.`
+			action
+				fail "<uhai> 4: Scan Geocoris"
+	npc
+		government "Hai"
+		to spawn
+			has "unfettered: entered Ya Hai"
+		personality launching coward
+		system "Ya Hai"
+		ship "Grasshopper (Ion)" "Beholder"
+	to fail
+		has "unfettered: outcast"
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog `You have not scanned the Geocoris yet. Go to Ya Hai to scan it, and then come back here.`
+	on complete
+		"unfettered: Solifuge deployment" ++
+		clear "unfettered: entered Ya Hai"
+		log `Scanned a Geocoris for the Unfettered. They seemed interested in the design - probably with a ship of their own in mind.`
+		payment 150000
+		dialog `You send the scans to the Unfettered government and receive a payment of <payment>. They tell you Mera will need some time to analyze the scans and you may return in a few weeks for further work.`
+		event "unfettered: analyzed the scans" 10 20
+
+mission "<uhai> 4: Geocoris Arrival"
+	landing
+	invisible
+	source
+		system "Ya Hai"
+	to offer
+		has "<uhai> 4: Scan Geocoris: active"
+	on offer
+		conversation
+			`Shortly after you land, you see the Geocoris signature on your radar. You take off immediately.`
+				flee
```

Comment:
Flee makes all ships in the system hostile to you, right? Shouldn’t this be `launch` or `depart`?
```suggestion
				launch
```

Reply:
Flee launches you and declines the mission, that's all (and there's no point in activating this one mission)

Reply:
Okay.